.github/workflows/ci-testing.yml

```diff
@@ -0,0 +1,46 @@
+# tox git actions integration was taken from:
+# https://github.com/ymyzk/tox-gh-actions/tree/e11a29e2e5c73b6e48f5d4ee379f903476d013c0
+# Tests unittest pass, documentation builds, and package builds
+name: Tests
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      matrix:
+        platform: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: [3.6, 3.7, 3.8]
```

Comment:
Just a thought, you might want to consider just avoiding dealing with 3.6.

Whilst MDA 2.0 is going to support 3.6->3.8 (maybe even 3.9), a large number of upstream packages are just about to drop 3.6 (e.g. numpy 1.20, matplotlib 3.4 [currently has a release candidate]). It's been making `pip` based installations of MDA a rail pain :(

Reply:
Perfect. I will remove that from the tests. Thanks

Reply:
Passing by after getting the commit notification. MDA 2.0 supports python 3.9 now, you'll probably want to add it in?

Reply:
perfect